package.json

```diff
@@ -28,6 +28,8 @@
     "libp2p": "0.42.2",
     "lru-cache": "7.14.1",
     "mnemonist": "0.39.5",
+    "multiformats": "^10.0.3",
```

Comment:
10 because this code already depends on it at that version but previously didn't declare that dep, so would use a random one of 3 different major versions that are in the dep tree. Version 11+ doesn't work with the test helpers due to https://github.com/multiformats/js-multiformats/blob/4a36fb7ee49edb4300267b90301ef0e4300cbc46/src/cid.js#L305